monkey/infection_monkey/ransomware/utils.py

```diff
@@ -0,0 +1,96 @@
+import os
+from typing import List
+
+VALID_FILE_EXTENSIONS_FOR_ENCRYPTION = {
```

Comment:
Should end up in the island probably. Not for this PR though.

Reply:
Why in the island?

Reply:
There's a chance we'd want to make them configurable.